x-pack/plugins/custom_branding/server/plugin.ts

```diff
@@ -48,14 +52,25 @@ export class CustomBrandingPlugin implements Plugin {
     const router = core.http.createRouter<CustomBrandingRequestHandlerContext>();
     registerRoutes(router);
 
-    const fetchFn = async (request: KibanaRequest): Promise<CustomBranding> => {
+    const fetchFn = async (
+      request: KibanaRequest,
+      unauthenticated?: boolean
+    ): Promise<CustomBranding> => {
+      if (!this.isValidLicense) {
+        return {};
+      }
```

Comment:
Do we have any test (unit or FTR) to asset that behavior?

Reply:
Not yet, since it's not being used yet.